x-pack/plugin/src/test/resources/rest-api-spec/test/ml/delete_forecast.yml

```diff
@@ -1,125 +1,10 @@
-setup:
```

Comment:
All these positive forecast cases were already covered in `org.elasticsearch.xpack.ml.integration.ForecastIT`